tests/functional/services/flows/test_flow_crud.py

```diff
@@ -23,12 +21,34 @@ def test_create_flow(flows_client, subscription_id):
         assert "subscription_id" not in req_body
 
 
+@pytest.mark.parametrize("value", [None, [], "dummy_value"])
```

Comment:
Both tests pass `"dummy_value"`, but the field is typed as `list[str]`.
I assume this is accidental, and `["dummy_value"]` was intended?

We aren't checking the types of parameters at runtime (and I don't think we necessarily should, at least at present), so the test works fine, but it would break if we ever decided to check in the future, so it's fragile if we're trying to test the normal operating modes for the function under test.